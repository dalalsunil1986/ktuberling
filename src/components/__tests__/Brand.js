import React from 'react';
import renderer from 'react-test-renderer';

import Brand from '../Brand';


describe('Brand', () => {
  it('render', () => {
    const tree = renderer.create(<Brand />);

    expect(tree).toMatchSnapshot();
  });
});
