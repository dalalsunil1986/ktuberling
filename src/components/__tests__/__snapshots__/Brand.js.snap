// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand render 1`] = `
<div
  className="navbar-header"
>
  <span
    className="navbar-brand"
  >
    <img
      alt="ktuberling"
      height="20"
      src="./assets/icon.png"
      width="20"
    />
  </span>
</div>
`;
